src/WCSimDetectorMessenger.cc

```diff
@@ -430,7 +432,11 @@ void WCSimDetectorMessenger::SetNewValue(G4UIcommand* command,G4String newValue)
                 } else if ( newValue == "nuPRISM_mPMT") {
 		  WCSimDetector->SetIsNuPrism(true);
 		  WCSimDetector->SetNuPrism_mPMTGeometry();
-                } else if ( newValue == "nuPRISMShort_mPMT") {
+                } else if ( newValue == "nuPRISMBeamTest_mPMT") {
+		WCSimDetector->SetIsNuPrism(true);
+		WCSimDetector->SetNuPrismBeamTest_mPMTGeometry();
+		} else if ( newValue == "nuPRISMShort_mPMT") {
+		WCSimDetector->SetIsNuPrismBeamTest(true); // Jun.04,2020 M.Shinoki
```

Comment:
Should this line actually go above?

Reply:
Thank you for your advice.
I fixed.

Reply:
great. please test compilation and run-time to make sure everything working as before.

Reply:
I checked that compilation is success and tested WCSim run briefly.
It seems to run correctly.